client/transactions/declarations.d.ts

```diff
@@ -103,6 +103,16 @@ declare module '@woocommerce/components' {
 }
 
 declare module '@woocommerce/navigation' {
+	import type { BrowserHistory, Location } from 'history';
+	/**
+	 * Extension of history.BrowserHistory but also adds { pathname: string } to the location object.
+	 */
+	interface WooBrowserHistory extends BrowserHistory {
```

Comment:
Creating this interface allows us to use the `getHistory` function with a correct return type.